man/fido2-crypttab.sh

```diff
@@ -10,3 +10,10 @@ sudo /usr/lib/systemd/systemd-cryptsetup attach mytest /dev/sdXn - fido2-device=
 # If that worked, let's now add the same line persistently to /etc/crypttab,
 # for the future.
 sudo bash -c 'echo "mytest /dev/sdXn - fido2-device=auto" >>/etc/crypttab'
+
+# Depending on your distribution you may need to manually regenerate your initramfs 
+# to be able to use a FIDO2-Device to unlock the partition at boot. (https://unix.stackexchange.com/a/705809)
+# On Fedora based systems:
+sudo dracut --regenerate-all --force
```

Comment:
`--regenerate-all` feels a bit risky. Maybe just regenerate the current kernel version, and tell users to boot into the same kernel version?

Reply:
I just removed the parameter. To be fair, I only really looked at the man page for dracut after creating this PR.
I would assume anyone actively using multiple kernel versions/variants (not just upgrading without thinking about it) should know this extra parameter and/or be able to adapt this command to their usecase. Also, recovery keys / passphrases are still usable when booting another version / kernel that has not yet been regenerated.